.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+> If you are reporting a bug, please use the following template.
+
+---
+
+### Bug Report
+Please answer the following questions for yourself before submitting an issue.
+
+- [ ] I checked the [documentation](http://dartsim.github.io/) and found no answer.
+- [ ] I checked to make sure that this issue has not already been filed.
+
+#### Environment
+
+* DART version: 6.3.0 / master
```

Comment:
I think my "outdated" [comment here](https://github.com/dartsim/dart/pull/943#discussion_r151265170) is still relevant.